src/app/lib/config/services/arabic.js

```diff
@@ -22,11 +22,11 @@ const service = {
     defaultImageAltText: 'BBC News عربي',
     dir: 'rtl',
     externalLinkText: '، خارجي',
-    imageCaptionOffscreenText: '،التعليق على الصورة',
-    videoCaptionOffscreenText: '،التعليق على الفيديو',
-    audioCaptionOffscreenText: '،التعليق على التسجيل السماعي',
+    imageCaptionOffscreenText: 'التعليق على الصورة، ',
```

Comment:
In the spread sheet the commas appear on the left and not on the right - might need confirmation though